src/orb.yml.hbs

```diff
@@ -848,42 +848,51 @@ commands:
           Only in use when verify-revision-is-deployed is set to true.
         type: boolean
         default: true
+      run-update-task-definition:
+        description: |
+          Whether to run the update-task-definition again in the update-service or
+          not. Defaults to false and will only run when parameter is set to true.
+        type: boolean
+        default: false
     steps:
-      - update-task-definition:
-          family: << parameters.family >>
-          container-image-name-updates: << parameters.container-image-name-updates >>
-          container-env-var-updates: << parameters.container-env-var-updates >>
-      - run:
-          name: Update service with registered task definition
-          command: |
-            DEPLOYMENT_CONTROLLER="$(echo << parameters.deployment-controller >>)"
+      - when:
```

Comment:
If we are going with renaming `run-update-task-definition` to `skip-task-definition-registration`, `when` should be updated accordingly to `unless` 🙂